tests/phpunit.xml.dist

```diff
@@ -8,17 +8,16 @@
       tests without defining these variables if you use the default values.
 -->
 <phpunit
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../vendor/phpunit/phpunit/phpunit.xsd"
     bootstrap="../vendor/autoload.php"
-    backupGlobals="false"
-    backupStaticAttributes="false"
-    syntaxCheck="false"
     colors="true">
   <testsuites>
     <testsuite name="unit">
-      <directory suffix="Test.php">unit</directory>
+      <directory>unit</directory>
```

Comment:
I'm not sure we want this change. Other projects won't be using our phpunit.xml.dist directly, will they? If someone copies this template to their own project, they may remove the suffix at that time if they do not want it.

For core Drush, I think we want to constrain that tests end in `Test.php`.

Reply:
OK, I can put it the suffix back. The only reason I removed it because it has a default value.
https://github.com/sebastianbergmann/phpunit/blob/6.5.14/phpunit.xsd#L164

Reply:
Oh, ok. In that case I am 👍 on taking it out.